lib/connectors/SentinelConnector/index.ts

```diff
@@ -37,15 +41,19 @@ export interface ISentinelConnectionOptions extends ITcpConnectionOptions {
   updateSentinels?: boolean
 }
 
+export interface ISentinelConnectorOptions extends ISentinelConnectionOptions {
+  sentinels: Partial<ISentinelAddress>[]
+}
```

Comment:
It seems `ISentinelConnectionOptions` isn't used other places inside the library except for `AsyncSentinelConnector`, which I think can be moved outside (like to the `examples` folder).

If people want to reuse `ISentinelConnectorOptions` with the `sentinels` property to be omitted, they can simply use `Pick<ISentinelConnectorOptions, Exclude<keyof TypeA, 'sentinels'>>`, so given that, I think we just need to keep one of the two interfaces.